src/components/CreateResetPassword/components/PasswordRequirementsList/PasswordRequirementsList.js

```diff
@@ -0,0 +1,84 @@
+import React, {
+  useState,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { Layout } from '@folio/stripes-components';
+
+import usePasswordRules from './queries';
+
+import styles from './PasswordRequirementsList.css';
+
+const passwordRequirementsNames = [
+  'password_length',
+  'numeric_symbol',
+  'special_character',
+  'alphabetical_letters',
+];
+
+const PasswordRequirementsList = ({ passwordValue }) => {
+  const [requiredRules, setRequiredRules] = useState([]);
+  const [unfulfilledRules, setUnfulfilledRules] = useState([]);
+
+  const { rules } = usePasswordRules();
+
+  useEffect(() => {
+    if (requiredRules.length <= 0 && rules?.length > 0) {
+      const requiredRulesSet = rules
+        .filter(rule => passwordRequirementsNames.includes(rule.name))
+        .map(rule => {
+          const splittedRuleDescription = rule.description.split(' must ');
+
+          rule.description = `Must ${splittedRuleDescription[1]}`;
```

Comment:
Please add a new translation with this message

Reply:
Fixed, thank you!